docs/DEVELOPER_GUIDE.md

```diff
@@ -333,5 +333,12 @@ Ideally, the `version` specified should match the release tag of the plugin.
 This helps users and maintainers to easily identify which version of the plugin
 they have installed.
 
+### Automating the releasing of new version
```

Comment:
## Automate releasing new versions on krew-index

Reply:
done

Reply:
The latest version I see still says

> Automating the releasing of new version

@rajatjindal Can you check again?

Reply:
yes, looks like I reverted that somehow, fixing again.